.travis/linux..install.sh

```diff
@@ -3,6 +3,7 @@
 PACKAGES="cmake libsndfile-dev fftw3-dev libvorbis-dev libogg-dev libmp3lame-dev
 	libasound2-dev libjack-jackd2-dev libsdl-dev libsamplerate0-dev libstk0-dev stk
 	libfluidsynth-dev portaudio19-dev g++-multilib libfltk1.3-dev
+	fluid
```

Comment:
I assume xenial doesn't provide `fluid` along side `libfluidsynth-dev`?  Can you please keep the formatting consistent and place it on an existing line?